lib/ex_doc/cli.ex

```diff
@@ -103,7 +103,7 @@ defmodule ExDoc.CLI do
   defp normalize_formatters(opts) do
     formatters =
       case Keyword.get_values(opts, :formatter) do
-        [] -> opts[:formatters] || ["html", "epub"]
+        [] -> opts[:formatters] || ["html", "epub", "markdown"]
```

Comment:
I don't think we shuold enable it by default :)

Reply:
Ah I misread and thought that was the allow-list of supported formatters